chapters/en/slides/module3/module3_03.Rmd

```diff
@@ -0,0 +1,245 @@
+to compare multiple distributions against each other.
+So far we only studied these plots with up to three different distributions to compare,
+but what if we had many more?
+
+---
+
+## The extended movies dataset
+
+```{python}
+import altair as alt
+from vega_datasets import data
+
+
+movies_extended = data.movies().dropna(subset=['Major Genre'])
+movies_extended
+```
+
+Notes:
+For this exercise we will use another movies dataset containing more genres.
+The question we will try to answer is
+"Which genres has the highest worldwide gross?".
+
+---
+
+## Many distributions can't be effectively compared with histograms
+
+```{python}
+alt.Chart(movies_extended).mark_bar().encode(
+    alt.X('Worldwide Gross', bin=alt.Bin(maxbins=30)),
+    alt.Y('count()'),
+    alt.Color('Major Genre'))
+```
+
+Notes:
+This visualization is not effective at all
+in answering our question.
+Most genres have a really low worldwide gross,
+and because our histogram is stacked
+this bar dominates with around 2000 movies,
+and extends the y-axis so that it is hard to see 
+how many observations there are for highest
+x-values
+(but we know there are some since the axis extends that far).
+
+---
+
+## Many distributions can't be effectively compared with densities either
+
+```{python}
+(alt.Chart(movies_extended).mark_area().transform_density(
+    'Worldwide Gross',
+    groupby=['Major Genre'],
+    as_=['Worldwide Gross', 'density'])
+ .encode(
+    alt.X('Worldwide Gross'),
+    alt.Y('density:Q'),
+    alt.Color('Major Genre')))
+```
+
+Notes:
+Although we saw previously that a layered density chart
+is better than a stacked histogram,
+it is no effective in comparing this many distributions
+and has many of the same problems as the histogram in the previous slide.
+
+So what can we do to create an effective comparison between all the genres?
+
+---
+
+## Bar charts are effective for comparing a single value but hides any variation
+
+```{python}
+alt.Chart(movies_extended).mark_bar().encode(
+    alt.X('mean(Worldwide Gross)'),
+    alt.Y('Major Genre'))
+```
+
+Notes:
+We could use a barplot if we were only interested in comparing a single value
+from each distribution,
+such as the mean or median.
+However,
+this is generally not a good idea because any plot that only shows a single value
+will hide the variation in our data
+and could lead to us drawing the wrong conclusion
+as you will see in the next slide.
+
+---
+
+## Showing a single value can lead to incorrect conclusions
+
+<img src=/module3/barplot-hiding-points.png></img>
+
+[Beyond Bar and Line Graphs: Time for a New Data Presentation Paradigm](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002128)
+
+Notes:
+This image illustrates how the same barplot (A)
+could have been generated from several different distributions (B, C, D, or E).
+If we just looked at the barplot,
+we could not tell which of the distributions it came from,
```

Comment:
Why is this important? It won't be clear to the students. It would be worthwhile here talking about the outlier in C and how that effects the mean, and how the mean (and thus our interpretation) would change if the outlier was removed (or we collected more data, of which very few would expected to be outliers). As well as talking about  D - in that maybe there are really two groups for the black bar group, and so we maybe should rethink our data and divide it into 3 groups?

Finally, for this slide, you should remove the error bars (the learners will have no idea what these are yet) and the statistical tests beneath, as again, they will have no meaning for these learners.

Reply:
I think we have highlighted why it is important in the second part of this sentence (we would arrive at a different conclusion regarding the data). I agree we can make it clearer exactly *how* the conclusions would change and will add info  in your example, thanks!

Agree to simplifying the image, I was thinking maybe making my own eventually but just pasted this for now. Altough it is CC BY 4.0, so we could just edit.